client-app/shared/account/components/add-points-operation-dialog.vue

```diff
@@ -0,0 +1,34 @@
+<template>
+  <VcPopup title="Add Operation">
+    <div class="px-10 py-10">
+      <form class="flex flex-col gap-5">
+        <VcInput v-model="reason" label="Reason" is-required></VcInput>
+        <VcInput v-model="amount" label="Amount" type="number" is-required></VcInput>
+      </form>
+    </div>
+    <template #actions="{ close }">
+      <VcButton is-submit class="uppercase px-5 py-5" @click="addPointsOperation()"> Add operation </VcButton>
```

Comment:
It's not needed to use parenthesis here, you can just use `@click="addPointsOperation"` instead.